zhaquirks/tuya/tuya_thermostat.py

```diff
@@ -116,6 +116,12 @@ def __init__(self, *args, **kwargs):
         )
         self.add_unsupported_attribute(Thermostat.AttributeDefs.pi_heating_demand.id)
 
+    async def write_attributes(self, attributes, manufacturer=None):
```

Comment:
I'm hesitant to make this change here, since we know `_TZE204_p3lqqy2r` is tested and working, though `manufacturer_id_override: t.uint16_t = foundation.ZCLHeader.NO_MANUFACTURER_ID` should have prevented the manufacturer id, this risks breaking that thermostat. I'd rather subclass TuyaThermostat, maybe something like this.

```python
class TuyaThermostatNoManuf(TuyaThermostat):
    """Tuya local thermostat cluster, no manufacturer id on attribute write."""

    async def write_attributes(self, attributes, manufacturer=None):
        """Overwrite to force manufacturer code."""
        return await super().write_attributes(
            attributes, manufacturer=foundation.ZCLHeader.NO_MANUFACTURER_ID
        )
```

Then just update the Avatto thermostats to use this cluster.